ui/src/app/applications/components/application-deployment-history/application-deployment-history-details.tsx

```diff
@@ -87,35 +166,60 @@ export const ApplicationDeploymentHistoryDetails = ({app, info, index}: props) =
-                                {(details: models.RepoAppDetails) => (
-                                    <div>
-                                        <ApplicationParameters
-                                            application={{
-                                                ...app,
-                                                spec: {...app.spec, source}
-                                            }}
-                                            details={details}
-                                        />
-                                    </div>
-                                )}
-                            </DataLoader>
+                        {showSourceDetails.includes(i) ? (
+                            <div
+                                id={`'history-expanded-'${i}`}
+                                key={`'history-expanded-'${i}`}
+                                className={classNames('white-box')}
+                                style={{
+                                    marginTop: '20px',
+                                    padding: '0px',
+                                    border: '2px solid rgba(0, 0, 0, 0.1)',
+                                    boxShadow: '1px 2px 3px rgba(0, 0, 0, 0.1)',
+                                    borderRadius: '4px'
+                                }}>
+                                <div id={`'history-expanded-'${i}`} key={`'history-expanded-'${i}`} className='white-box__details' style={{marginBottom: '0px'}}>
+                                    {getExpandedSection(i)}
+                                    <DataLoader
+                                        errorRenderer={err => {
+                                            return getErrorSection(err);
+                                        }}
+                                        input={{
+                                            ...source,
+                                            targetRevision: recentDeployments[index].revisions[i],
+                                            index: i,
+                                            versionId: recentDeployments[index].id,
+                                            appName: app.metadata.name
+                                        }}
+                                        load={src => services.repos.appDetails(src, src.appName, app.spec.project, i, recentDeployments[index].id)}>
+                                        {(details: models.RepoAppDetails) => (
+                                            <React.Fragment>
+                                                <div
+                                                    id={'floating_title_' + i}
+                                                    className='editable-panel__sticky-title'>
+                                                    <div style={{marginTop: '20px'}}>
```

Comment:
```suggestion
                                        {(details: models.RepoAppDetails) => (
                                            <React.Fragment>
                                                <div id={'floating_title_' + i} className='editable-panel__sticky-title'>
                                                    <div style={{marginTop: '20px'}}>
```

Linter doesn't like multi-line divs

Reply:
Thanks for making these changes. To make things ever-so-slightly-simpler for me, so I don't have to pull down your commit and merge it with my new changes, I just made the change in my workspace.